elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -47,12 +47,17 @@ RUN yum -y update \
     && yum install -y unzip glibc.x86_64 cmake \
     && yum clean all
 
-RUN groupadd -g 1000 elasticsearch && \
-    adduser -u 1000 -g 1000 -d /usr/share/elasticsearch elasticsearch
+ARG DEFAULT_UID=1000
+ARG DEFAULT_GID=1000
+ENV DEFAULT_UID $DEFAULT_UID
+ENV DEFAULT_GID $DEFAULT_GID
```

Comment:
DEFAULT_UID and DEFAULT_GID are environment variables, overridable by docker build-time arguments, set to 1000.